ha/const.py

```diff
@@ -49,9 +49,15 @@
 SOURCE_CLI_SCHEMA_FILE = "{}/cli_schema.json".format(SOURCE_CONFIG_PATH)
 CLI_SCHEMA_FILE = "{}/cli_schema.json".format(CONFIG_DIR)
 
+# IEM DESCRIPTION string: To be removed
```

Comment:
I will remove this after the IEM schema is available through alert generator tasks.